### PR TITLE
HTS-1581: Removed duplicate h1 and title values

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: app/uk.gov.hmrc.helptosavestridefrontend/views/customer_not_eligible.scala.html [hunk 1]
```diff
@@ -50,7 +51,7 @@ <h2 class="heading-medium error-summary-heading" id="error-summary-give-nino">@m
         }
     }
 
-    <h1 class="heading-large">@messages("hts.eligibility.not.eligible.h1")</h1>
+    <h1 class="heading-large">@messages("hts.eligibility.not.eligible.title.h1")</h1>
```
Comment (sophie-still) [inline app/uk.gov.hmrc.helptosavestridefrontend/views/customer_not_eligible.scala.html]: Can change to @title on line 54

File: app/uk.gov.hmrc.helptosavestridefrontend/views/account_created.scala.html [hunk 2]
```diff
@@ -19,12 +19,14 @@
 
 @(accountNumber: String)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
 
+@title = @{Messages("hts.account-created.title")}
```
Comment (sophie-still) [inline app/uk.gov.hmrc.helptosavestridefrontend/views/account_created.scala.html]: hts.account-created.title.h1

File: app/uk.gov.hmrc.helptosavestridefrontend/views/enter_customer_details.scala.html [hunk 3]
```diff
@@ -32,13 +32,15 @@
 
 @(nino: String, form: Form[ApplicantDetails])(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
 
+@title = @{messages("hts.customer-eligible.enter-details.title")}
```
Comment (sophie-still) [inline app/uk.gov.hmrc.helptosavestridefrontend/views/enter_customer_details.scala.html]: .h1

File: conf/messages [hunk 4]
```diff
@@ -120,8 +117,7 @@ hts.create.account.terms.ul1-li3-unknown=they will tell us each time they leave
 hts.create.account.terms.ul1-li4=they are creating a Help to Save account for the first time
 hts.create.account.terms.p2=Ask the customer if they understand and agree to the terms and conditions, and want to create a Help to Save account.
 
-hts.account.exists.title=Customer already has an account
-hts.account.exists.h1=Customer already has an account
+hts.account.exists.title.h1=Customer already has an account
 hts.account.exists.p1=Customer can pay in by bank transfer or standing order. They will need to set up the payment through their bank.
 
 hts.account-created.title=Help to Save account created
```
Comment (sophie-still) [inline conf/messages]: hts.account-created.title will change with the above comment

File: app/uk.gov.hmrc.helptosavestridefrontend/views/account_created.scala.html (hunk 2 above)
Comment (Jackie1988) [reply app/uk.gov.hmrc.helptosavestridefrontend/views/account_created.scala.html]: done, thanks

File: app/uk.gov.hmrc.helptosavestridefrontend/views/customer_not_eligible.scala.html (hunk 1 above)
Comment (Jackie1988) [reply app/uk.gov.hmrc.helptosavestridefrontend/views/customer_not_eligible.scala.html]: done, thanks

File: app/uk.gov.hmrc.helptosavestridefrontend/views/enter_customer_details.scala.html (hunk 3 above)
Comment (Jackie1988) [reply app/uk.gov.hmrc.helptosavestridefrontend/views/enter_customer_details.scala.html]: done

File: conf/messages (hunk 4 above)
Comment (Jackie1988) [reply conf/messages]: yep

**Pull request merged**